assemblies/assembly_about-rhdh.adoc

```diff
@@ -0,0 +1,19 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-10-31
```

Comment:
Is this required?

Reply:
Thank you, @hmanwani-rh. The newdoc version and the templated generated date were created automatically when I added the assembly. Looking at previous assemblies, we can do without them. Would you recommend that I remove them?